tests/model_serving/maas_billing/multitenancy/utils.py

```
@@ -53,6 +53,8 @@
 TENANT_GATEWAY_ROUTE_SUFFIX = "-route"
 GATEWAY_ACCESS_LABEL_PREFIX = "maas.opendatahub.io/gateway-access-"
 GATEWAY_HTTP_PORT = 80
+GATEWAY_HTTPS_PORT = 443
+GATEWAY_TLS_CERTIFICATE_SECRET_NAME = "data-science-gateway-service-tls"  # pragma: allowlist secret
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "GATEWAY_TLS_CERTIFICATE_SECRET_NAME"

(S105)